starlark/testdata/math.star

```diff
@@ -0,0 +1,43 @@
+# Tests of math module.
+
+load('math.star', 'math')
+load('assert.star', 'assert')
+
+def assert_near(actual, desired, threshold):
+  val = desired - actual
+  if val < 0:
+    val = val * -1
+  return val < threshold
+
+assert.eq(math.ceil(0.5), 1.0)
```

Comment:
The tests should demonstrate the behavior of each function more thoroughly: does it accept int as well as float? What if the argument is 0, 0.4, 0.5, 1, 10, inf, nan, or the negation of each? What is the round mode?